tests/test_notifications.py

```diff
@@ -262,7 +261,7 @@ def test_node_subscriptions_and_backrefs_removed_when_node_is_deleted(self):
             subscription.save()
 
             s = getattr(project.creator, 'email_transactional', [])
-            assert_equal(len(s), 1)
+            assert_equal(len(s), 2)
```

Comment:
Why does it have an `email_transactional` attribute? That's not [explicitly mentioned](https://github.com/mfraezz/osf.io/blob/711558a79268f0b72630083b3c2a70d97875fe9e/tests/factories.py#L74-L136).